ISSUES.txt

```diff
@@ -1,365 +0,0 @@
-
```

Comment:
So this was silently dropped with the 3.0.1 release? Are there things in there which are still current and for which we should open issues?